rfcs/text/0030/file_description.yml

```diff
@@ -0,0 +1,178 @@
+# Licensed to Elasticsearch B.V. under one or more contributor
+# license agreements. See the NOTICE file distributed with
+# this work for additional information regarding copyright
+# ownership. Elasticsearch B.V. licenses this file to you under
+# the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# 	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+---
+- name: file_description
+  title: File description
+  group: 2
+  short: These fields contain information about a file descriptor.
```

Comment:
"information about a file".  (not file descriptor, thats just the integer/handle).

you may want to add: "Linux uses the file abstraction for files, pipes, fifos, network sockets and more."